README.md

```diff
@@ -242,6 +242,8 @@ Docker EE is on the same code base as Docker CE, so also built from Moby, with c
 * [docker-bench-security](https://github.com/docker/docker-bench-security) - script that checks for dozens of common best-practices around deploying Docker containers in production. By [@docker][docker]
 * [notary](https://github.com/docker/notary) - a server and a client for running and interacting with trusted collections. By [@docker][docker]
 * [oscap-docker](https://github.com/OpenSCAP/openscap) - OpenSCAP provides oscap-docker tool which is used to scan Docker containers and images. By RedHat
+* [Sysdig Falco](https://github.com/draios/falco) - Sysdig Falco is an open source container security monitor. It can monitor application, container, host, and network activity and alert on unauthorized activity.
+* [Sysdig Secure](https://www.sysdig.com/product/secure) :heavy_dollar_sign: - Sysdig Secure addresses run-time security through behavioral monitoring and defense, and provides deep forensics based on open source Sysdig for incident response.
```

Comment:
Sorry but there is a redirect for this URL. Could you replace it like `https://www.sysdig.com/product/secure  → https://sysdig.com/product/secure/`